python/cudf/cudf/tests/test_json.py

```diff
@@ -278,7 +278,7 @@ def test_json_lines_byte_range(json_input):
 )
 def test_json_lines_dtypes(json_input, dtype):
     df = cudf.read_json(json_input, lines=True, dtype=dtype)
-    assert all(df.dtypes == ["float32", "int32", "int16"])
+    assert all(df.dtypes == ["float64", "int64", "int16"])
```

Comment:
why this change?

Reply:
So until now if the user passed `"float"` or `"int"`, csv reader would map them to `"float32"` or `"int32"` respectively. But that is not the general convention with `cudf`/`numpy`/`pandas`, `int` translates to `int64` and `float` translates to `float64`. 

```python
>>> import numpy as np
>>> np.dtype('float')
dtype('float64')
>>> np.dtype('int')
dtype('int64')
```

Reply:
now I'm even more glad to remove the string-based dtype API from libcudf :) This would be either inconsistent or counterintuitive there.

Reply:
Yup, removing string based dtype selection surfaced this issue :tada: